Modules/Core/Common/test/itkStreamingImageFilterTest3.cxx

```diff
@@ -42,13 +42,9 @@ itkStreamingImageFilterTest3(int argc, char * argv[])
   using PixelType = unsigned char;
   using ImageType = itk::Image<PixelType, 2>;
 
-  using ReaderType = itk::ImageFileReader<ImageType>;
-  ReaderType::Pointer reader = ReaderType::New();
-  reader->SetFileName(inputFilename);
-
   using SomeFilter = itk::ShiftScaleImageFilter<ImageType, ImageType>;
   SomeFilter::Pointer filter = SomeFilter::New();
-  filter->SetInput(reader->GetOutput());
+  filter->SetInput(itk::ReadImage<ImageType>(inputFilename));
```

Comment:
This is a change in the behavior of this test. This test is called "streaming" because it test streaming. This change update the whole image from a file then runs the filter.

Reply:
OK, thanks Bradley, but would that be a problem? The test still passes successfully, right?

Reply:
The test passes successfully, but its whole point is to test streaming. I did not catch this in my review.

Reply:
OK, I will remove the proposed itkStreamingImageFilterTest3 changes from this PR, with the next force-push. Thanks @blowekamp and @dzenanz

Reply:
Thanks @N-Dekker